### PR TITLE
Fix vector map

### PR DESCRIPTION
**Pull request by hakuturu583**
add node data in vector_map export tools.

### CODE REVIEW

File: Assets/Scripts/Map/MapUtil.cs [hunk 1]
```diff
@@ -819,6 +819,30 @@ public static Vector MakeVector(int VID, int PID, double Hang, double Vang)
...
+                    NID = 1,
+                    PID = 1,
+                };
+            }
+
+            public static Node MakeNode(int NID, int PID)
+            {
+                return new Node()
+                {
+                    NID = NID,
```
Comment (andre-rosa) [inline Assets/Scripts/Map/MapUtil.cs]: Are the parameters shadowing the Node fields NID/PID?

**Comment by hadiTab** [comment]
@hakuturu583 can you sign the CLA? Thanks.

File: Assets/Scripts/Map/MapUtil.cs (hunk 1 above)
Comment (hakuturu583) [reply Assets/Scripts/Map/MapUtil.cs]: > @hakuturu583 can you sign the CLA? Thanks.
Excuse me, what is CLA?

**Comment by hakuturu583** [comment]
> @hakuturu583 can you sign the CLA? Thanks.

Excuse me what is CLA??

**Comment by seonman** [comment]
> > @hakuturu583 can you sign the CLA? Thanks.
> 
> Excuse me what is CLA??

It's Contributor License Agreement (https://en.wikipedia.org/wiki/Contributor_License_Agreement ). It shows that a contributor is allowed to make a contribution and that the project has the right to distribute it. As you can see in the link, many open source projects requires contributors to sign CLA when they are contributing code to the projects.

**Comment by hakuturu583** [comment]
Thank you!! I signed!!

**Comment by hadiTab** [comment]
We've received the signed CLA on our end. 
Thanks for submitting the pull request. We will be including your commits in our next update.

**Pull request closed without merging**